enterprise/cluster/src/test/java/org/neo4j/cluster/protocol/election/ElectionContextTest.java

```diff
@@ -188,14 +189,52 @@ public void testElectionNotOkQuorumFailedFiveInstances()
     }
 
     @Test
-    public void twoVotesFromSameInstanceForSameRoleShouldBeConsolidated() throws Exception
+    public void testInstanceWithLowestIdFailedIsNotConsideredTheElector()
+    {
+        Set<InstanceId> failed = new HashSet<>();
+        failed.add( new InstanceId( 1 ) );
+        failed.add( new InstanceId( 2 ) );
+
+        Map<InstanceId, URI> members = new HashMap<>();
+        members.put( new InstanceId( 1 ), URI.create( "server1" ) );
+        members.put( new InstanceId( 2 ), URI.create( "server2" ) );
+        members.put( new InstanceId( 3 ), URI.create( "server3" ) );
+        members.put( new InstanceId( 4 ), URI.create( "server4" ) );
+        members.put( new InstanceId( 5 ), URI.create( "server5" ) );
+
+        Config config = mock( Config.class );
+        when( config.get( ClusterSettings.max_acceptors ) ).thenReturn( 10 );
+
+        ClusterConfiguration clusterConfiguration = mock( ClusterConfiguration.class );
+        when( clusterConfiguration.getMembers() ).thenReturn( members );
+        when( clusterConfiguration.getMemberIds() ).thenReturn( members.keySet() );
+
+        ClusterContext clusterContext = mock( ClusterContext.class );
+        when( clusterContext.getConfiguration() ).thenReturn( clusterConfiguration );
+
+        MultiPaxosContext context = new MultiPaxosContext( new InstanceId(3 ), Iterables.iterable(
+                new ElectionRole( "coordinator" ) ), clusterConfiguration,
+                mock( Executor.class ), NullLogProvider.getInstance(),
+                mock( ObjectInputStreamFactory.class ), mock( ObjectOutputStreamFactory.class ),
+                mock( AcceptorInstanceStore.class ), mock( Timeouts.class ),
+                mock( ElectionCredentialsProvider.class), config );
+
+        context.getHeartbeatContext().getFailed().addAll( failed );
```

Comment:
Maybe standard //when comment on this?

Maybe also another sanity-check assertion just before this just to check that we were not the elector before we were informed about the failed instances?